css/index.css

```diff
@@ -0,0 +1,157 @@
+html{
+    font-family: 'Playfair Display', serif;
+}
+
+body{
+    color: #262626;
+    background-color: 242, 242, 242;
+    margin: 52px;
+    padding: 22px;
+    font-size: 15px;
+}
+
+/* header and navigation links */
+header{
+    font-size: 22px;
+    color: #ff751a;
+    overflow: hidden;
+}
+header li{
+    display: inline;
+    color: #ff751a;
+    align-items: right;
+}
+#menu ul{
+    text-align: center;
+    float: right;
+    margin: none;
+}
+
+/* headings */
+h2{
+    font-weight: bold;
+    font-size: 22px;
+    color: #ff7733;
+}
+h3{
+    font-size: 17px;
+    font-weight: bold;
+    color: #737373;
+}
+body{
+    font-size: 16px;
+}
```

Comment:
This is a duplicate of line 10, it's overwriting the value there. Just update line 10 to `16px` instead